query/src/main/scala/filodb/query/exec/PromQlRemoteExec.scala

```diff
@@ -44,12 +43,12 @@ case class PromQlRemoteExec(queryEndpoint: String,
   override val urlParams = Map("query" -> promQlQueryParams.promQl)
   private val dummyQueryStats = QueryStats()
 
-  override def sendHttpRequest(execPlan2Span: Span, httpTimeoutMs: Long)
-                              (implicit sched: Scheduler): Future[QueryResponse] = {
+  override def sendRequest(execPlan2Span: Span, httpTimeoutMs: Long)
+                          (implicit sched: Scheduler): Task[QueryResponse] = {
 
     import PromCirceSupport._
     import io.circe.parser
-    remoteExecHttpClient.httpPost(queryEndpoint, requestTimeoutMs,
+    val fut = remoteExecHttpClient.httpPost(queryEndpoint, requestTimeoutMs,
```

Comment:
nit: more meaningful var name, "future" maybe?

Reply:
These are cherry-picked commits -- I'll leave these as-is